### ISSUE TITLE
Compositions not always correct

### ISSUE THREAD

**Issue by koeleboe** [iRail/iRail#518]
For example 

https://api.irail.be/v1/composition/?id=4137 says there is one 1 desiro, but the realtime nmbs says 2 desiros.
![Screenshot_20240502_165137_Samsung Internet](https://github.com/iRail/iRail/assets/72651779/29fce407-1a0b-4bb3-866c-622863f2f14a)

**Comment by Bertware** [comment]
Testing locally shows the correct composition for both trains, and in both cases the source has changed as well. 
Seems like the composition was updated 2024-05-02 17:01:03, cache duration should probably be decreased on compositions in order to get these updates

**Comment by koeleboe** [comment]
Yeah, that makes sense, but weird that the composition is updated after the train arrived at the last station.
